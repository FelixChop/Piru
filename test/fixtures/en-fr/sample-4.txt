She was over the moon when she finally cracked the enigmatic puzzle that had baffled her for weeks.
