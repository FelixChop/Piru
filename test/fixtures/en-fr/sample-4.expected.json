[
  {
    "word": "over the moon",
    "lemma": "over the moon",
    "definition": "très heureux",
    "citation": "She was over the moon when she finally cracked the enigmatic puzzle that had baffled her for weeks.",
    "to_learn": true
  },
  {
    "word": "baffled",
    "lemma": "to baffle",
    "definition": "déconcerter",
    "citation": "She was over the moon when she finally cracked the enigmatic puzzle that had baffled her for weeks.",
    "to_learn": true
  }
]
